bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/i18n/sagercaster_fr.properties

```diff
@@ -44,7 +44,7 @@ beaufortLabel = Beaufort
 beaufortDescription = Force du vent mesurée sur l'échelle Beaufort
 pressureDescription = Pression barométrique au niveau de la mer.
 timestampChannelLabel = Horodatage
-timestampChannelDescription = Horodatage de la derni�re mise � jour de la pr�vision m�t�o.
+timestampChannelDescription = Horodatage de la derni?re mise ? jour de la pr?vision m?t?o.
```

Comment:
Something is wrong here.

Reply:
Yes I was already fixing it.

Reply:
Oups, I just fixed it in Crowdin. Don't know what will happen now :-(

Reply:
I pressed the sync button so we will shortly see.

Reply:
The issue was introduced by @clinique in https://github.com/openhab/openhab-addons/pull/11665. I think his editor is not using UTF-8 so it messes up the files by using different encodings.

Reply:
It could have been spotted in the review where you can see the old translations having weird characters:

![Screenshot from 2021-12-05 11-09-42](https://user-images.githubusercontent.com/12213581/144742252-576b31e3-08ea-441b-b1fc-d8c55649e358.png)

Reply:
I let you merge this PR when this problem is solved.
This is OK for me for the French changes.